python/snippets/mov_avg.py

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env python3
+
+import os
+import time
+from collections import deque
+from typing import Deque
+
+import grpc
+from remotivelabs.broker.sync import SignalWrapper, BrokerException, Client as BrokerClient
+
+# Let´s keep the last window of signals here
+received_signals: Deque[SignalWrapper] = deque(maxlen=int(os.environ.get("WINDOW", 1000)))
+subscription = None
+
+try:
+
+    client = BrokerClient()
+    client.on_connect = lambda c: print(f"Yay! - connected as {c.client_id} to {c.url}")
+    client.on_signals = received_signals.extend  # Add all signals to received_signals
+
+    client.connect(url=os.environ.get("URL", "http://localhost:50051"),
+                   api_key=os.environ.get("API_KEY", None))
```

Comment:
```suggestion
    client = BrokerClient.connect(
                   url=os.environ.get("URL", "http://localhost:50051"),
                   on_connect=lambda c: print(f"Yay! - connected as {c.client_id} to {c.url}"),
                   on_signals = received_signals.extend  # Add all signals to received_signals
                   api_key=os.environ.get("API_KEY", None))
```

As part of comment in API PR. To avoid having `BrokerClient` in two states: connected vs. not connected.

Reply:
why?

Reply:
@JohanKarlbergg thought on this?